pom.xml

```diff
@@ -2060,6 +2060,8 @@
           </executions>
           <configuration>
             <scalaVersion>${scala.version}</scalaVersion>
+            <checkMultipleScalaVersions>true</checkMultipleScalaVersions>
```

Comment:
Also not strictly related, but helpful when I was debugging another Java 11 issue.